versioned_docs/version-2.0/concepts/evaluation/evaluation.mdx

```diff
@@ -381,3 +381,39 @@ See our LangSmith video series to go deeper on these concepts:
 | Accuracy  | Standard definition             | No (ground truth class) | No                                                                   | No                |
 | Precision | Standard definition             | No (ground truth class) | No                                                                   | No                |
 | Recall    | Standard definition             | No (ground truth class) | No                                                                   | No                |
+
+## Deciding which evaluation approach to use
```

Comment:
nit: needs standardize usage of gerund vs verb (see `Deciding` here vs `Choose` below)